contributors/guide/contributor-cheatsheet/README-de.md

```diff
@@ -0,0 +1,393 @@
+# Cheat-Sheet für Kubernetes-Beitragende
+
+[Bahasa Indonesia](README-id.md) | [日本語](README-ja.md) | [한국어](README-ko.md) | [Português](README-pt.md) | [中文](README-zh.md)
```

Comment:
This can be removed from here. :)

Related: can you add german to this line in the main README?

Reply:
Sure!